src/frontend/src/btc/components/convert/BtcConvertReview.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import { Collapsible } from '@dfinity/gix-components';
+	import BtcConvertFeeTotal from '$btc/components/convert/BtcConvertFeeTotal.svelte';
+	import BtcConvertFees from '$btc/components/convert/BtcConvertFees.svelte';
+	import ConvertReview from '$lib/components/convert/ConvertReview.svelte';
+	import MessageBox from '$lib/components/ui/MessageBox.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { OptionAmount } from '$lib/types/send';
+
+	export let sendAmount: OptionAmount;
+	export let receiveAmount: number | undefined;
+</script>
+
+<ConvertReview on:icConvert on:icBack {sendAmount} {receiveAmount}>
+	<Collapsible slot="fee">
+		<!-- The width of the item below should be 100% - collapsible expand button width (1.5rem) -->
+		<div class="flex w-[calc(100%-1.5rem)] items-center" slot="header">
+			<BtcConvertFeeTotal />
+		</div>
+
+		<BtcConvertFees {sendAmount} />
+	</Collapsible>
+
+	<div slot="info-message" class="mt-4">
+		<MessageBox>{$i18n.convert.text.conversion_may_take}</MessageBox>
+	</div>
+
+	<slot name="cancel" slot="cancel" />
+</ConvertReview>
+
+<style lang="scss">
+	:global(div.contents > div.header > button.collapsible-expand-icon) {
```

Comment:
You're right, I updated it and now it's scoped to the current component. 

And yes, unfortunately, it's needed - the expand icon container take 3rem when the icon itself is just half of that, and by default it uses `justify-content: flex-start`. If I don't override it, the button is then not aligned with the other items of the list:

<img width="557" alt="Screenshot 2024-11-18 at 16 33 21" src="https://github.com/user-attachments/assets/c5785fba-d2c7-4bf9-9ba1-c4a87a7f7d13">

Reply:
It cannot be solved by wrapping a div around something? Always try to avoid to use `:global` as much as possible.

Reply:
Just tried once again - the button is deep inside the Collapsible component, no access to it from outside. The only way here would be to use `externalToggle` option and implement the button on our side (including state to decide which icon to show) which is also far from ideal. I would proceed with the CSS :global approach here, unless you think that the other one is still preferable.

Reply:
We can have a global but, we should ultimately resolve this. If it needs an extra variable in gix-cmp,let's add it.

Reply:
That said. We should not have a `:global` at the top, it should be at least scoped.

Reply:
Mmm did you push?

```
<style lang="scss">
	:global(.btc-convert-review-fees > div.contents > div.header > button.collapsible-expand-icon) {
		justify-content: flex-end;
	}
</style>
```

Reply:
Yes, to make it "scoped", I added `.btc-convert-review-fees` to bind this CSS rule just to this case. The keyword `:global` should still stay, otherwise CSS won't be applied to children components due to the way how Svelte treats "local" styles.

Reply:
Alright. It would be really nice to resolve this afterwards.